showcase/app/controllers/components/table.js

```diff
@@ -438,4 +466,9 @@ export default class ComponentsTableController extends Controller {
       user.isAnimated = false;
     });
   }
+
+  @action
+  noop() {
```

Comment:
might be a good idea to extract into simple helper

Reply:
I agree. We have a lot of repetition throughout our tests/examples that can probably be condensed down into a reusable toolkit. I think that's out of scope for this PR, but I'm going to bring it up in our next engineering sync for discussion. Thanks for the input!